ibis/backends/duckdb/registry.py

```diff
@@ -144,6 +144,15 @@ def _regex_extract(t, op):
     return result
 
 
+def _json_get_item(t, op):
```

Comment:
This change turned up a bug in duckdb's JSON extract operator implementation (https://github.com/duckdb/duckdb/issues/5063), which only shows up for queries where the `path` provided to `->` is parametrized AND there are no other parameters in the query. Previously this was masked since an implicit limit was always present, which would provide a 2nd parameter, avoiding the bug.

Reply:
Ah, fun. Nice find.